<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElencoTari</class>
 <widget class="QDialog" name="ElencoTari">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>736</width>
    <height>517</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>736</width>
    <height>517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Elenco utenze TARI</string>
  </property>
  <widget class="QPushButton" name="btnElencoEsci">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Esci</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBtnElencoAziende">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Azienda</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBtnElencoPrivato">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Privato</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblElencoTari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>711</width>
     <height>381</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QComboBox" name="comboBoxLettera">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>H</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>J</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>O</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>R</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>U</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>W</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Z</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblLettera">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostra elenco per lettera:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerca">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
